MAINTAINERS.md

```diff
@@ -5,3 +5,4 @@ TJBot was lovingly created by researchers at IBM Research.
 - [Maryam Ashoori](https://github.com/maryamashoori) is the creator of TJBot and is the ringleader of our merry band of cardboard robots.
 - [Justin Weisz](https://github.com/jweisz) created the TJBot library and generally tries to bring order to our chaos.
 - [Victor Dibia](https://github.com/victordibia) created TJBot’s initial receipes and is our master of demos.
+- [Bruno Braga](https://github.com/bgbraga) created TJBot bootstrap and helping to maintain the project.
```

Comment:
Im not sure the entire content of this document is the right way to report attribution. 
I would leave it unedited and subject for a different PR when we have had a chance to discuss this on slack.